invenio_rdm_records/services/pids/providers/base.py

```diff
@@ -211,7 +212,7 @@ def validate(
                 )
                 return False, [
                     _(
-                        f"{self.pid_type}:{identifier} is already registered" +
+                        "{self.pid_type}:{identifier} is already registered" +
```

Comment:
I think this won't work...I think the goal is to replace f strings with the `'{myvar}'.format(myvar='foo')` format....

Reply:
yes, i just got it.